src/Provider/Wurfl.php

```diff
@@ -57,6 +57,66 @@ public function getParser()
     }
 
     /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isRealBrand($value)
+    {
+        if ($value === '' || $value === null) {
+            return false;
+        }
+
+        if ($value == 'Generic') {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isRealModel($value)
+    {
+        if ($value === '' || $value === null) {
+            return false;
+        }
+
+        if (substr((string) $value, 0, 7) == 'Android') {
```

Comment:
Do type casting next time before the  `if` for all at once , so it's easiert to read/understand